pkg/apis/submariner.io/v1/types.go

```diff
@@ -77,10 +79,20 @@ type GatewayStatus struct {
 	Version       string       `json:"version"`
 	HAStatus      HAStatus     `json:"haStatus"`
 	LocalEndpoint EndpointSpec `json:"localEndpoint"`
+	// +optional
+	Latency       LatencySpec  `json:"latency"`
 	StatusFailure string       `json:"statusFailure"`
 	Connections   []Connection `json:"connections"`
 }
 
+type LatencySpec struct {
+	Last    string `json:"last"`
+	Min     string `json:"min"`
+	Average string `json:"average"`
+	Max     string `json:"max"`
+	Mdev    string `json:"mdev"`
```

Comment:
What is this one? Can we use a more descriptive name?

Reply:
I changed the names of the field, do you think it makes more sense now ? or should I  add comments?

Reply:
"StdDev" looks OK.